docs/source/quick_start/getting_started_with_tensorbay.rst

```diff
@@ -46,9 +46,6 @@ Authorize a Client Instance
       :start-after: """Authorize a Client Instance"""
```

Comment:
this change needs no scope in commit message,
which means remove the `(quick_start)` in commit message.

Reply:
Also better to not use abbreviation in commit message, which means use `delete` rather than `del`